sirbot/slack/database/sqlite/user.py

```diff
@@ -10,8 +10,16 @@
          (id, dm_id, admin, raw, last_update)
          VALUES (?, ?, ?, ?, ?)''',
```

Comment:
you also need to add the column name and a `?` in values for the `deleted` property